docs/topics/upgrade.md

```diff
@@ -100,3 +100,24 @@ There are known limitations with VMSS cluster-autoscaler scenarios and upgrade.
 ### Cluster-autoscaler + Availability Set
 
 We don't recommend using `aks-engine upgrade` on clusters that have Availability Set (non-VMSS) agent pools `cluster-autoscaler` at this time.
+
+## Forcing an upgrade
+
+The upgrade operation takes an optional `--force` argument :
+
+```shellscript
+-f, --force   
+force upgrading the cluster to desired version. Allows same version upgrades and downgrades.
+```
+
+In some situation, you might want to bypass the aks-engine validation on your api-model versions and cluster nodes versions. This is at your own risks and you should assess the potential harm of using this flag.
+
+The `--force` parameter instructs the upgrade process to :
+
+- ignore normal version validation
+- include __all__ your cluster's nodes (masters and agents) in the upgrade process. 
+- It will __not__ skip nodes that are already on the target version.
+- It will allow any kubernetes versions, including the ones that have not been whitelisted, or deprecated.
+- It will accept downgrade operations.
+
+For each nodes, the cluster will follow the same process as described in the section above: [Under the hood](#under-the-hood)
```

Comment:
Oh, now I get it. I think the reference `#under-the-hood` will work without you changing anything about the header above.